modules/packages/snap

```diff
@@ -5,7 +5,7 @@ command=$1
 # Output maybe contain backslashes, and we don't want those to end up escaping
 # something so we use use -r with read.
 while read -r line; do
-  export INPUT_$line
+  INPUT_$line
```

Comment:
This will try to execute a command like `INPUT_File=/some/file` instead of creating such variable.

Reply:
Well, in *bash*, I don't know which shell this module is supposed to use.

Reply:
`declare` instead of `export` would work better.

Reply:
ack

Reply:
mmm, i dont have declare

Reply:
restored export since declare didn't work.

```
> $ ./modules/packages/snap get-package-data <<EOF
heredoc> File=syncthing
heredoc> EOF
./modules/packages/snap: 55: ./modules/packages/snap: declare: not found
```

Reply:
@vpodzime wat?
```
% bash
$ INPUT_File=/some/file
$ echo $INPUT_File
/some/file
```

Reply:
```

Reply:
```
$ cat read_input.sh 
#!/bin/bash

while read -r line; do
  declare INPUT_$line
done

echo "$INPUT_File"
$ bash read_input.sh 
File=some/file
some/file
```